### ISSUE TITLE
docs: Documentation error? - Port for Unifi Controller - 8443 or 443

### ISSUE THREAD

**Issue by daviesap** [unifi-poller/unifi-poller#306]
Hi

I have just got unifi-poller up and running in a Docker container - all good.

But I followed the instructions, and specified port 8443 for the Unifi controller (https://10.0.15.101:8443).  This failed.  Changing it to 443 worked.

**Comment by davidnewhall** [comment]
Yup. it's 443 on the UDM! (and I think the cloudkey now too) 8443 for most other folks. I'll add some more data to the docs/config file to call this out.

**Comment by daviesap** [comment]
Thanks!

I am on a Cloud Key Gen 2 - with firmware 2.0.26 running the new(?) Unifi OS

Thanks for all
Andrew

**Comment by mellow129** [comment]
I just took the update tonight:
UniFi-CloudKey-Gen2-Plus, Firmware | 2.0.27
Unifi Network Controller, Current Version 6.0.45 (Build: atag_6.0.45_14358)


The update deleted my unifi-poller account. I had to re-add it in the CloudKey users area, and I had to change the port from 8443 to 443 in the up.conf to establish communication. I set the new account to local admin, unifi network read only.

Hope this helps someone.
